docs/modules/ROOT/pages/s2s-transferring-erc20.adoc

```diff
@@ -12,14 +12,14 @@ Once you have completed step 1 to setup ERC20 tokens with another chain, end-use
 
 The following one-time setup is required for SKALE Chains with a default access control policy (default settings are: whitelisting enabled, automatic deployment disabled). For more information on IMA access control, xref:access-control.adoc[see here].
 
-=== 1. Enable Automatic Deploy
+=== 1. Connect each SKALE Chain
 
-include::partial$autodeploy-note.adoc[]
+⇢ Add the `target` chain using `connectSchain` within the **TokenManagerLinker** contract on the `origin` chain. 
 
-First, enable automatic deploy on the `origin` chain.
+⇠ To exit from the target, you must also reciprocate the connection by adding the `origin` chain using `connectSchain` on the `target` chain.
 
 [NOTE]
-Only the `AUTOMATIC_DEPLOYER_ROLE` may enable or disable automatic deployment.
+Only the `REGISTRAR_ROLE` may execute `connectSchain`.
```

Comment:
REGISTRAR_ROLE in TokenManagerLinker. So you need to be granted with this role on this contract